src/libos/src/net/socket/unix/addr.rs

```diff
@@ -86,7 +86,7 @@ impl Addr {
         addr.sun_family = AddressFamily::LOCAL as libc::sa_family_t;
 
         let addr_len = match self {
-            Self::File(unix_path) => {
+            Self::File(unix_path, inode_num) => {
```

Comment:
the second parameter is useless, why not mark it with `Self::File(unix_path, _)`

Reply:
Done.